### PR TITLE
[enhancement](test) exclude somecases not support in cloud mode

### PR DESCRIPTION
**Pull request by dataroaring**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

**Comment by dataroaring** [comment]
run buildall

File: regression-test/pipeline/cloud_p0/conf/regression-conf-custom.groovy [hunk 1]
```diff
@@ -17,7 +17,7 @@
...
-excludeSuites = "test_index_failure_injection,test_dump_image,test_profile,test_spark_load,test_refresh_mtmv,test_bitmap_filter,test_information_schema_external,test_stream_load_new_move_memtable,test_stream_load_move_memtable,test_materialized_view_move_memtable,test_disable_move_memtable,test_insert_move_memtable,set_and_unset_variable,test_pk_uk_case_cluster,test_point_query_cluster_key,test_compaction_uniq_cluster_keys_with_delete,test_compaction_uniq_keys_cluster_key,test_set_partition_version,test_show_transaction"
+excludeSuites = "test_index_failure_injection,test_dump_image,test_profile,test_spark_load,test_refresh_mtmv,test_bitmap_filter,test_information_schema_external,test_stream_load_new_move_memtable,test_stream_load_move_memtable,test_materialized_view_move_memtable,test_disable_move_memtable,test_insert_move_memtable,set_and_unset_variable,test_pk_uk_case_cluster,test_point_query_cluster_key,test_compaction_uniq_cluster_keys_with_delete,test_compaction_uniq_keys_cluster_key,test_set_partition_version,test_show_transaction,test_be_inject_publish_txn_fail,test_report_version_missing,test_publish_timeout,set_replica_status"
```
Comment (yujun777) [inline regression-test/pipeline/cloud_p0/conf/regression-conf-custom.groovy:20]: split excludeSuites into multi lines:  
···
excludeSuites = """ 
   xx,
   xx
"""
···

**Review by yujun777 — COMMENTED** [review]
need update

**Review by gavinchou — APPROVED** [review]

**Review by liaoxin01 — APPROVED** [review]

**Pull request merged**